cmd/kpod/load.go

```diff
@@ -95,14 +95,18 @@ func loadCmd(c *cli.Context) error {
 		output = os.Stdout
 	}
 
+	options := libpod.CopyOptions{
+		SignaturePolicyPath: c.String("signature-policy"),
+	}
+
 	src := libpod.DockerArchive + ":" + input
-	if err := runtime.PullImage(src, false, c.String("signature-policy"), output); err != nil {
+	if err := runtime.PullImage(src, options, output); err != nil {
 		src = libpod.OCIArchive + ":" + input
 		// generate full src name with specified image:tag
 		if image != "" {
 			src = src + ":" + image
 		}
-		if err := runtime.PullImage(src, false, "", output); err != nil {
+		if err := runtime.PullImage(src, options, output); err != nil {
```

Comment:
Should output be moved to the cp options?

Reply:
I don't see why not. Will move it.

Reply:
fixed